sandbox/messages/2bBBpsiMl2rnQt99qm6JLZ6w1vq.json

```diff
@@ -0,0 +1,49 @@
+{
+  "data": {
+    "type": "Message",
+    "id": "2bBBpsiMl2rnQt99qm6JLZ6w1vq",
+    "attributes": {
+      "routingPlan": {
+        "id" : "b838b13c-f98c-4def-93f0-515d4e4f4ee1",
+        "version" : "1"
+      },
+      "messageReference": "1226417c-f683-4d68-afe8-2d52321a6b70",
+      "messageStatus": "delivered",
+      "messageStatusDescription": "Contact detail is missing",
+      "channels": [
+        {
+          "type": "sms",
+          "retryCount": 0,
+          "channelStatus": "delivered",
+          "channelStatusDescription": "Contact detail is missing",
+          "timestamps": {
+            "created": "2023-10-09T10:31:59Z",
+            "updated": "2023-10-09T10:52:12Z",
```

Comment:
`updated` timestamp shouldn't be present - this looks like it was copied from an old example message - see previous comment about making sure we're rebased on `release`